src/mixxx.cpp

```diff
@@ -219,7 +219,6 @@ void MixxxMainWindow::initialize(QApplication* pApp, const CmdlineArgs& args) {
         pConfig->getValue(ConfigKey("[Controls]", "Tooltips"),
                 static_cast<int>(mixxx::TooltipsPreference::TOOLTIPS_ON)));
 
-    setAttribute(Qt::WA_AcceptTouchEvents);
```

Comment:
I am concerned this hack may prevent us from implementing custom touch gestures. However, we currently have a major bug to fix so we can defer that issue.

Reply:
Any widget can enable touch events individually. 

This tells Qt that the MixxxMainWindow itself has a event handler that accept touch events. 
This hack was required in early Qt versions to detect that no child widget has consumed the touch and we could resend it as mouse click. Now Qt calculated this forehead, which disables the mouse click synthesizer.